bundles/action/org.openhab.action.ciscospark/src/main/java/org/openhab/action/ciscospark/internal/CiscoSpark.java

```diff
@@ -0,0 +1,158 @@
+/**
+ * Copyright (c) 2010-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.action.ciscospark.internal;
+
+import java.util.UUID;
+
+import javax.mail.internet.AddressException;
+import javax.mail.internet.InternetAddress;
+
+import org.openhab.core.scriptengine.action.ActionDoc;
+import org.openhab.core.scriptengine.action.ParamDoc;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.ciscospark.Message;
+import com.ciscospark.Spark;
+import com.ciscospark.SparkException;
+
+/**
+ * This class provides static methods that can be used in automation rules for
+ * sending Messages via Cisco Spark.
+ *
+ * @author Tom Deckers
+ * @since 1.9.0
+ */
+public class CiscoSpark {
+
+    private static final Logger logger = LoggerFactory.getLogger(CiscoSpark.class);
```

Comment:
@9037568 Loggers in nearly all other OH1 add-ons are static, so I think it's OK for this to be as well.  If it ever comes time to run through this repo and change all loggers to non-static, it's the difference between changing 1,000 and 1,001 cases.  Code in openhab2-addons, by contrast, started out using non-static loggers so it's required in that repo.